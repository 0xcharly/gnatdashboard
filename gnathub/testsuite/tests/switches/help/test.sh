gnathub --help | grep "^Usage:"
gnathub --help | grep "GNATHUB.ERRORS"
