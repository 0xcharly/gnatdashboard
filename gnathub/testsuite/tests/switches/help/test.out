Usage: gnathub [-vq] -P PROJECT [--plugins PLUGINS | --exec SCRIPT]
